mne/io/base.py

```diff
@@ -1997,6 +2004,16 @@ def _write_raw(fname, raw, info, picks, fmt, data_type, reset_range, start,
 
         pos_prev = pos
 
+    if raw.annotations is not None:
+        start_block(fid, FIFF.FIFFB_MNE_ANNOTATIONS)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MIN, raw.annotations.onset)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MAX, raw.annotations.duration)
+        write_name_list(fid, FIFF.FIFF_COMMENT, raw.annotations.description,
+                        separator=';')
```

Comment:
Yeah, but to me the default (colon) is something that would surely see regularly in the annotation descriptions. I thought semicolon would be safer. Annotations class has a sanity checker for semicolon now.